universum/initializer.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+from .modules.output.output import MinimalOut
+from .lib.gravity import Module
+from .lib.module_arguments import ModuleArgumentParser
+
+__all__ = ["Initializer"]
+
+
+class Initializer(Module):
+    description: str = "Create a dummy project"
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.out = MinimalOut()
+
+    def execute(self) -> None:
+        target_dir = "universum_dummy_project"
```

Comment:
I thought we agreed to create file in current directory.

Reply:
I thought we agreed to create an example source directory in current directory, to be than replaced with actual source directory. In my opinion it's more obvious and is easier to clean afterwards.